examples/python/Basic/half_edge_mesh.ipynb

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
LGTM after moving.

We've restructured our documentation. Please merge `master` to this branch and move this file to`examples/python/geometry/half_edge_mesh.ipynb`